buildspec.yml

```diff
@@ -3,22 +3,29 @@ version: 0.2
 phases:
   install:
     commands:
-      - echo "deb http://ppa.launchpad.net/ubuntu-toolchain-r/test/ubuntu trusty main" > /etc/apt/sources.list.d/toolchain.list
+      - add-apt-repository ppa:ubuntu-toolchain-r/test -y
```

Comment:
Is `add-apt-repository` natively available, without any further installation steps?

Reply:
Yes; furthermore, this is already the case on the underlying Ubuntu trusty; it's not a tweak in the docker image.